API/src/db.js

```diff
@@ -2,7 +2,7 @@ var connection_info;
 try {
   connection_info = require('./connection_info');
 } catch (err) {
-  connection_info = process.env.DATABASE_URL;
+  connection_info = 'postgres://cryst:Spirals@localhost:5432/cryst';//process.env.DATABASE_URL;
```

Comment:
don't push your changes to this file. Instead, make a connection_info file in the same folder and put your connection info in there. The connection_info file should be included in the git ignore already so it will not be pushed.